analysis/config/deployment.yaml

```diff
@@ -19,7 +19,7 @@ spec:
         image: ko://github.com/ossf/package-feeds/analysis
         env:
           - name: OSSF_MALWARE_ANALYSIS_RESULTS
-            value: gs://ossf-malware-analysis-results
+            value: gs://naveen-analysis
```

Comment:
This is me mucking around to make sure it is able to deploy it. This will not be committed.